### ISSUE TITLE
Make Puzzle Mode Work With Other Rendering Options

### ISSUE THREAD

**Issue by tsj845** [radicise/territopple#33]
Puzzle mode does not work with the other rendering options, this needs to be fixed.

**Comment by tsj845** [comment]
puzzles are being deprecated
